package.json

```diff
@@ -21,12 +21,13 @@
   ],
   "license": "MIT",
   "peerDependencies": {
-    "eslint": ">= 8.23.0"
+    "eslint": ">= 9.10.0"
   },
   "dependencies": {
-    "eslint-plugin-n": "^17.9.0",
-    "@typescript-eslint/eslint-plugin": "^7.15.0",
-    "@typescript-eslint/parser": "^7.15.0"
+    "@eslint/js": "^9.10.0",
+    "eslint-plugin-n": "^17.10.3",
+    "globals": "^15.9.0",
+    "typescript-eslint": "^8.6.0"
```

Comment:
Since we are creating new presets, it is a good chance for us to add a few more presets. E.g. we could add https://www.npmjs.com/package/eslint-plugin-mocha since almost all Hexo repos use mocha.

Reply:
need `"type": "module",` or name js file to mjs

Reply:
Then keep using `const = require` then. In the ESLint world, CommonJS is still the first-class citizen。